aospy_synthetic/test/test_db.py

```diff
@@ -3,78 +3,98 @@
 import unittest
 from subprocess import call
 
-from test_objs import (cases, models, example_projects, variables,
-                       regions)
-from aospy_synthetic.calc import Calc, CalcInterface
+from test_objs import (runs, models, projects, variables,
+                       regions, calc_objs)
 from aospy_synthetic.db.sqlalchemy.sqlalchemy_db import SQLAlchemyDB
-#from aospy_synthetic.SQLAlchemyDB import SQLBackend
+from aospy_synthetic.db.sqlalchemy.sqlalchemy_config import (ProjDB, ModelDB,
```

Comment:
This just occurred to me...will it be the case that every backend will have to implement DB objects at each object level, as done here?  That may not be answerable, because I suppose it depends on how the backend and coders approach things.

But, for the sake of argument, if these can be expected to be universal, then maybe we could create abstractions of them too?  Do you think that would be useful?